cart.css

```diff
@@ -0,0 +1,82 @@
+@import url(/global-styles.css);
+
+.main{
+    background-color: var(--off-white);
+    height: 100vh;
+}
+
+.cart-body {
+    display: flex;
+    gap: 5rem;
+    justify-content: space-around;
+}
+.cart-title {
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding: 1rem;
+}
+
+.card {
+    width: 20rem;
+    border: 1px solid var(--clr-black);
+}
+
+.card-title{
+    text-align: center;
+}
+.card-sub-title{
+    text-align: center;
+    
+}
```

Comment:
Suggestion: To make code cleaner, do it like this:

```
.card-title,
.card-sub-title {
    text-align: center;
}
```

Better yet, use utility class for `text-align: center` and apply it wherever needed.